src/nemo_safe_synthesizer/data_processing/dataset.py

```
@@ -62,24 +64,20 @@ def _handle_enum_value(v: object) -> None | int | float | bool | str:
         # types will be truncated by int(v), so check equality to make sure
         # we haven't lost precision.
         t = int(v)  # ty: ignore[invalid-argument-type] -- third-party stub mismatch
         if t == v:
             return t
-    except Exception:
-        pass
 
     try:
         # Convert to python float if possible
         return float(v)  # ty: ignore[invalid-argument-type] -- third-party stub mismatch
     except Exception:
```

Ruff (0.15.13)

[warning] Do not catch blind exception: `Exception`

(BLE001)